pkg/glusterfs/controllerserver.go

```diff
@@ -342,7 +342,7 @@ func (cs *ControllerServer) ValidateVolumeCapabilities(ctx context.Context, req
 	resp := &csi.ValidateVolumeCapabilitiesResponse{
 		Supported: capSupport,
 	}
-	glog.V(1).Infof("GlusterFS CSI driver volume capabilities: %v", resp)
+	glog.V(1).Infof("glusterfs CSI driver volume capabilities: %+v", resp)
```

Comment:
Probably want to keep "GlusterFS" as a proper name to be consistent w/ usage like L196.
... also present a couple other places in this commit

Reply:
@JohnStrunk done. Replaced to GlusterFS